code/admin/CommentAdmin.php

```diff
@@ -48,36 +48,46 @@ public function getEditForm($id = null, $fields = null) {
 
 		$commentsConfig = CommentsGridFieldConfig::create();
 
-		$needs = new GridField(
-			'Comments', 
-			_t('CommentsAdmin.NeedsModeration', 'Needs Moderation'), 
-			Comment::get()->filter('Moderated',0),
+		$new = new GridField(
+			'NewComments',
+			_t('CommentsAdmin.NewComments', 'New Comments'),
+			Comment::get()->filter('IsModerated', 0),
```

Comment:
Good call on the `Moderated` column name. I'm a dummy sometimes!

@sminnee and the team have some feedback on the arrangement and naming of tabs; so this is likely to change before I'm done with the PR. At the moment spam and non-spam are in the same (first) list.